packages/synthetic-chain/cli.ts

```diff
@@ -20,7 +21,9 @@ const { positionals, values } = parseArgs({
   allowPositionals: true,
 });
 
-const allProposals = readProposals(path.resolve('.'));
+const proposalsParent = path.resolve('.');
```

Comment:
this isn't inaccurate but I think more importantly it's the root of the testing project.
```suggestion
const root = path.resolve('.');
```